spec/twilio_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'twilio.rb'
+
+describe TwilioSend do
+
+  # xit "should use Twilio to send a text message" do
+  # end
+
+
+  #   twilio_client = mock(Twilio::REST::Client)
+  #   expect(subject).to receive(:sms)
+  #
+
+  end
```

Comment:
Lint/BlockAlignment: `end` at 13, 2 is not aligned with `describe TwilioSend do` at 3, 0